File: azure/repind-ri-gui-cs.old/src/app/shared/models/common/last-activity.ts
/**
 * Class last activity: Last modification according to the given gin
 */
export class LastActivity {

    constructor(private _gin: string, private _dateModification: Date, private _sourceModification: string, private _signatureModification: string, private _siteModification:string) 
    {  }

 
    get gin(): string
    {
        return this._gin;
    }

    get dateModification(): Date 
    {
        return this._dateModification;
    }

    get sourceModification(): string 
    {
        return this._sourceModification;
    }

    get signatureModification(): string 
    {
        return this._signatureModification;
    }
    get siteModification(): string 
    {
        return this._siteModification;
    }

}

File: azure/repind-repind/msv/cf/graphql-bff-example/src/main/java/com/afklm/repind/msv/graphql/bff/example/model/LastActivity.java
package com.afklm.repind.msv.graphql.bff.example.model;

import jakarta.validation.constraints.NotBlank;
import lombok.*;

import java.util.Date;

/**
 * Last activity model
 */
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class LastActivity {


    /**
     * Description: Individual GIN number, example: '110000038701'
     */
    @NotBlank
    private String gin;
    /**
     * Description: last modification date, example: "04/10/22"
     */
    @NotBlank
    private Date dateModification;

    /**
     * Description: Site where modification originated, example: "Individuals_all"
     */
    private String sourceModification;
    /**
     * Description: Signature for individual modification, example: "REPIND/IHM"
     */
    private String signatureModification;

    /**
     * Description: Site where modification originated, example: "QUALIF2009"
     */
    private String siteModification;

}
